web/plugs/JWTverifier.ex

```diff
@@ -0,0 +1,59 @@
+require IEx
+defmodule Janitor.Plugs.JWTVerifier do
```

Comment:
@Xypus What If we just give it more straightforward name like "UserAuthorized" ?